.evergreen/config_generator/components/compile_only.py

```diff
@@ -61,6 +61,10 @@ def generate_tasks():
             if any(pattern in distro_name for pattern in ['power8', 'zseries']):
                 patchable = False
 
+            # etc/calc_release_version.py: error: unknown option `--format=...'
```

Comment:
```suggestion
            # etc/calc_release_version.py: error: unknown option `format...'
```

To match [current message](https://parsley.mongodb.com/evergreen/mongo_cxx_driver_compile_only_matrix_compile_only_rhel79_release_shared_impls_patch_dd0e48bdada5f3c3128791199352a4bb83dfb0c2_679bde77f496dd00075708ab_25_01_30_20_18_00/0/task?bookmarks=0,131&shareLine=67).